interpreter/cling/lib/Interpreter/IncrementalExecutor.h

```diff
@@ -44,6 +45,7 @@ namespace llvm {
 }
 
 namespace cling {
+  class DynamicLibraryManager;
```

Comment:
You have the `#include` already, and you need the `#include`.

Reply:
Did you forget to push? Also https://github.com/root-project/root/pull/6606/commits/186f48de9a21fd561c87221c47f5f8c552d8755f#r513248946 is marked resolved without any visible change in the sources?